db/migrate/20170103144759_encrypt_user_credentials.rb

```diff
@@ -0,0 +1,6 @@
+class EncryptUserCredentials < ActiveRecord::Migration[5.0]
+  def change
+      rename_column :user_credentials, :credential_value, :encrypted_credential_value
```

Comment:
Yea, it's the migrating from no encryption to encryption that I'm not sure how to handle. Do we think enough users want this feature?